src/main/java/org/prebid/cache/metrics/MetricsRecorder.java

```diff
@@ -6,14 +6,15 @@
 public abstract class MetricsRecorder
 {
     public enum MeasurementTag {
-        REQUEST_DURATION("request_duration"),
-        REQUEST_RATE("request_rate"),
-        ERROR_RATE("error_rate"),
-        RESOURCE_NOT_FOUND_RATE("resource-not-found_rate"),
-        BAD_REQUEST_RATE("bad-request_rate"),
-        INVALID_REQUEST_RATE("invalid-request_rate"),
-        JSON_REQUEST_RATE("json-request_rate"),
-        XML_REQUEST_RATE("xml-request_rate");
+        REQUEST_DURATION("pbc.prefix.request.duration"),
+        REQUEST_RATE("pbc.prefix.request"),
```

Comment:
Now that we got rid of `_rate` suffix in metric name it's better to remove `_RATE` suffix from enum names as well to avoid confusion. This applies to names following too.